### ISSUE TITLE
Comment appears to be causing an error

### ISSUE THREAD

**Issue by j-bernardi** [domdomegg/gdoc2latex#21]
```bash
gdoc2latex -i myFile.html
Error: Unsupported tag sup with content:
	\hyperref[id:cmnt1]{[a]}
Hint: View help with --help
```

I can't upload my file here, but here's some snippets with redactions:
```html
<body class="c83 c90 doc-content">
<p class="c39 c99 title" id="h.t62n5dang9x3">
<span class="c87">
My title
</span>
</p>
<sup>
<a href="#cmnt1" id="cmnt_ref1">[a]</a></sup><sup><a href="#cmnt2" id="cmnt_ref2">[b]</a>

</sup>
<p class="c20">
<span> Another piece of text</span><span>30 Aug 2024</span>
```

And at the bottom
```html
<div class="c37">
<p class="c25">
<a href="#cmnt_ref1" id="cmnt1">[a]</a>
<span class="c3">older version here:</span>
</p>
<p class="c25 c51">
<span class="c3">
</span>
</p>
<p class="c25">
<span class="c3">linkToOlderVersion</span>
</p>
</div>
<div class="c37">
<p class="c25">
<a href="#cmnt_ref2" id="cmnt2">[b]</a>
<span class="c3">Comment text in response to link</span>
</p>
<p class="c25">
<span class="c3">One total reaction</span>
</p>
<p class="c25">
<span class="c3">Jamie Bernardi reacted with &#129761; at 2024-09-02 08:20 am
</span>
</p>
</div>
```
Workaround: I deleted everything in my file included between `<sup>` and `</sup>`, and it worked :)

**Comment by domdomegg** [comment]
I've added a test for comments, but I don't seem to be able to replicate this: https://github.com/domdomegg/gdoc2latex/commit/5e1f51c7e2318b304cc2edc43be732cb953c4cb1

Do you have any other information on what might be causing an error here?

**Comment by domdomegg** [comment]
Thanks for sending the extra info by email! Fixed in https://github.com/domdomegg/gdoc2latex/commit/c40fb8409d43b13f6540381d2db68c68a5b436ce, released in [5.0.1](https://github.com/domdomegg/gdoc2latex/releases/tag/v5.0.1)
